sass/package.json

```diff
@@ -1,5 +1,7 @@
 {
   "devDependencies": {
+    "@types/node": "^11.13.8",
```

Comment:
Why do we need this if we aren't depending on the TS compiler?

Reply:
You are right, deleted.